src/ImageSharp.Drawing.Paths/PointInfoExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="PointInfoExtensions.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Drawing
+{
+    /// <summary>
+    /// Extension methods for helping to bridge Shaper2D and ImageSharp primitives.
+    /// </summary>
+    internal static class PointInfoExtensions
+    {
+        /// <summary>
+        /// Converts a <see cref="SixLabors.Shapes.PointInfo"/> to an ImageSharp <see cref="PointInfo"/>.
+        /// </summary>
+        /// <param name="source">The source.</param>
+        /// <returns>A <see cref="PointInfo"/> representation of this <see cref="SixLabors.Shapes.PointInfo"/></returns>
+        public static PointInfo Convert(this SixLabors.Shapes.PointInfo source)
```

Comment:
can't as the `PointInfo` we are converting to is inside `ImageSharp.Drawing` the one we are converting from is `SixLabors.Shapes` and the only project that knows about both sides is  `ImageSharp.Drawing.Paths`